tests/validation/psram/ci.json

```diff
@@ -1,4 +1,18 @@
 {
+  "soc_tags": {
+    "esp32": [
+      "psram"
+    ],
+    "esp32s2": [
+      "psram"
+    ],
+    "esp32s3": [
+      "octal_psram"
+    ],
+    "esp32c5": [
+      "psram"
+    ]
+  },
```

Comment:
p4?

Reply:
The generic runner for P4 already has psram. There are no p4 runners with the psram tag.